doc/source/user_guide/intro.rst

```diff
@@ -11,6 +11,120 @@ nekRS.
 Solving in Dimensional vs Non-Dimensional Form
 ----------------------------------------------
 
+It is often advantageous to solve these equations in non-dimensional form
+because non-dimensional formulation can simplify the simulation input parameters
+and provide more numerical stability. Here, we introduce the non-dimensional
+form in as general a manner as possible, assuming the use of variable material
+properties for density, viscosity, specific heat capacity, and thermal
+conductivity that are functions of temperature:
+:math:`\rho=\rho(T)`, :math:`\mu=\mu(T)`, :math:`C_p=C_p(T)`, and :math:`k=k(T)`.
+For simplicity, the functional notation is omitted throughout. 
+:ref:`Table 1 <characteristic_quantities>` provides some basic parameters for 
+nondimensionalization in nekRS.
+
+.. table:: Characteristic quantities for non-dimensionalization
+  :name:   characteristic_quantities
+
+  ================== ============================ ================================================== 
+  Symbol             Physical meaning             In NekRS           
+  ================== ============================ ================================================== 
+  :math:`D`          Characteristic length        Hydraulic diameter                
+  :math:`U`          Characteristic velocity      Reference velocity                          
+  :math:`\rho_0`     Characteristic density       Reference density
+  :math:`T_0`        Characteristic temperature   Reference temperature                           
+  :math:`\Delta T`   Characteristic velocity      Reference temperature rise relative to :math:`T_0`
+  ================== ============================ ==================================================
+
+Below are the nondimensional parameter in reference to table 
+:ref:`characteristic_quantities`.
+
+.. math::
+
+  x^* = \frac{x}{D}, \quad u^* = \frac{u}{U},
+  \quad t^* = \frac{tU}{D}, \quad P^* = \frac{P}{\rho_0 U^2}, 
+  \quad T^* = \frac{T - T_0}{\Delta T}, \quad  \rho^* = \frac{\rho}{\rho_0}
+
+Here, a subscript of * denotes nondimensionalized parameter, a subscript of 0
+indicates that the property is evaluated at :math:`T_0`. Inserting these
+non-dimensional variables into the mass and momentum conservation equations gives: 
+
+.. math:: 
+  
+  \rho^* \rho_0 \left(\frac{\partial(u^* U)}{\partial(t^* (D/U))}+\frac{u^* U}{D}\cdot\nabla u^* U\right)=-\frac{1}{D} \nabla(P^* (\rho_0 U^2))+\frac{1}{D} \nabla\cdot\mu\left{\frac{1}{D} (\nabla(u^* U)+\nabla(u^* U)^T )\right}+\rho f
```

Comment:
This formula isn't rendering correctly for me locally or on the readthedocs [build](https://nekrs-jezsw.readthedocs.io/en/18-introduction-dimensional-vs-non-dimensional-form/user_guide/intro.html#solving-in-dimensional-vs-non-dimensional-form). @Tri2008 could you take a look?